src/cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -1698,88 +1708,114 @@ msgstr "Revision"
 msgid "Show revisions"
 msgstr "Revisionen anzeigen"
 
-#: templates/pages/page_form.html:175 templates/pages/page_revisions.html:66
+#: templates/pages/page_form.html:160
+#: templates/pages/page_tree_archived_node.html:25
+msgid "Implicitly archived by parent page"
+msgstr "Implizit archiviert durch übergeordnete Seite"
```

Comment:
While this message is technically 100% correct, I'm not sure if users will understand the meaning. Do we need to make this more descriptive? Something like "A parent of this page is archived, therefore this page is also archived."

Reply:
I wasn't sure about this, too. I thought it might help region managers to understand what's going on if we use the wording explicitly/implicitly archived also in the UI. This message in particular should be the shortest possible representation of the status (where other pages got "Public", "In Review", or "Archived"). Do you have an idea for an equally short message?

Apart from that, there are also more descriptive messages in other parts of the template:

---------------------

![Screenshot_2020-12-02 Integreat Backend](https://user-images.githubusercontent.com/16021269/100871158-182dd280-34a0-11eb-83c4-7b34861955d9.png)

---------------------

![Screenshot_2020-12-02 Integreat Backend(2)](https://user-images.githubusercontent.com/16021269/100871159-182dd280-34a0-11eb-9cd9-4f4d3ac398c3.png)

Reply:
The shortest possible alternative I can come up with right now is "A parent is archived therefore this page is archived".

Reply:
@svenseeberg Do you think that the new wording is easier to understand?

Reply:
@svenseeberg ping